gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -287,6 +287,43 @@ case class TransformPreOverrides(isAdaptiveContext: Boolean)
       throw new UnsupportedOperationException(s"${other.getClass.toString} is not supported.")
   }
 
+  private def processProjectExecTransformer(
+      project: ProjectExecTransformer): ProjectExecTransformer = {
+    // special treatment for Project containing all bloom filters:
+    // if more than one bloom filter, Spark will merge them into a named_struct
+    // and then broadcast. The named_struct looks like {'bloomFilter',BF1,'bloomFilter',BF2},
+    // with two bloom filters sharing same nam. This will cause problem for some backends,
+    // e.g. ClickHouse, which cannot tolerate duplicate type names in struct type
+    // So we need to rename 'bloomFilter' to make them unique.
+    if (project.projectList.size == 1) {
```

Comment:
Since this part mainly focuses on generating new project list, can we put this part inside `ProjectExecTransformer`? That could help keep `ColumnarOverrides` concise.